Core/src/main/java/com/paypal/android/core/APIClient.kt

```diff
@@ -0,0 +1,18 @@
+package com.paypal.android.core
+
+class APIClient(private val configuration: PaymentsConfiguration) {
+
+    private val http = Http()
+    private val httpRequestFactory = HttpRequestFactory()
```

Comment:
Can we inject `http` and `httpRequestFactory` in the constructor? It should make passing instances much easier in our unit tests.